scripts/scil_filter_tractogram_anatomically.py

```diff
@@ -83,6 +92,71 @@ def _build_arg_parser():
     return p
 
 
+def binarize_labels(atlas, label_list):
+    #Input atlas must be a numpy array
+    mask = np.zeros(atlas.shape, dtype=np.uint16)
+    for label in label_list:
+        bin = atlas == label
+        mask[bin] = 1
+
+    return mask
+
+
+def dilate_mask(mask, mask_shape, vox_size, radius):
```

Comment:
If the argument is a (binary) mask, as it happens to be if I read the code correctly, a single call to `scipy.ndimage.binary_dilation` should suffice. And that would be a single line, so we would not need a method to do this. Maybe the bash script was dilating the labels themselves instead of the binary masks.

Reply:
I took a look at the binary_dilation method. The dilation seems to work great with binary arrays, but how can I take into account the voxel size when I want to apply a dilation of a certain radius (which might be smaller/bigger than the voxel dimension)? Could I be able to take that into account with calling with a specific structure element? Maybe I'm mistaken but I think the method I wrote does a little bit more, and I don't how to translate it!

Reply:
After discussing it with Florence, my approach was maybe too naïve, as I was thinking in voxels (and hence integer-valued radii) instead of physical space values, which we are aiming at here. So the comment can be disregarded.

Reply:
Done in dbd0b2a.